program/src/execute_sale/mod.rs

```diff
@@ -79,28 +82,15 @@ pub struct ExecuteSale<'info> {
             metadata.key().as_ref(),
             reward_center.key().as_ref()
         ],
-        bump = offer.bump
+        bump = offer.bump,
+        close = buyer,
     )]
     pub offer: Box<Account<'info, Offer>>,
 
     /// Payer account for initializing purchase_receipt_account
     #[account(mut)]
     pub payer: Signer<'info>,
 
-    /// The purchase ticket that would be initialized to record an executed sale
-    #[account(
-        init_if_needed,
-        space = PurchaseTicket::size(),
-        payer = payer,
-        seeds = [
-            PURCHASE_TICKET.as_bytes(),
-            listing.key().as_ref(),
-            offer.key().as_ref(),
-        ],
-        bump,
-    )]
-    pub purchase_ticket: Box<Account<'info, PurchaseTicket>>,
-
```

Comment:
Removing purchase tickets from program and will instead index execute_sale instructions and write similar record into the holaplex index.